docs/source/_auto_api_templates/python/attribute.rst

```diff
@@ -0,0 +1,60 @@
+{% if obj.display %}
```

Comment:
This is to get the docstrings for `hyperparameter_ranges` etc to show up

Reply:
My comment is probably related to this :d

Reply:
Good catch with the original comment. I just pushed up a change to fix this! This is how it looks locally now:

![image](https://user-images.githubusercontent.com/41651716/124655651-20068880-de6e-11eb-845f-915f09e0754a.png)

Made the same change for the linear regressor!